barretenberg/cpp/src/barretenberg/commitment_schemes/shplonk/shplemini.test.cpp

```diff
@@ -221,4 +222,122 @@ TYPED_TEST(ShpleminiTest, CorrectnessOfGeminiClaimBatching)
 
     EXPECT_EQ(shplemini_result, expected_result);
 }
+
+/**
+ * @brief Libra masking univariates are used in sumcheck to prevent the leakage of witness data through the evaluations
+ * of round univariates. Here we test the opening of log_n Libra masking univariates batched with the opening of several
+ * prover polynomials and their shifts.
+ *
+ */
+TYPED_TEST(ShpleminiTest, ShpleminiWithMaskingLibraUnivariates)
+{
+    using ShpleminiProver = ShpleminiProver_<TypeParam>;
+    using ShpleminiVerifier = ShpleminiVerifier_<TypeParam>;
+    using KZG = KZG<TypeParam>;
+    using IPA = IPA<TypeParam>;
+    using Fr = typename TypeParam::ScalarField;
+    using Commitment = typename TypeParam::AffineElement;
+    using Polynomial = typename bb::Polynomial<Fr>;
+
+    const size_t n = 16;
+    const size_t log_n = 4;
+    const size_t LIBRA_UNIVARIATE_LENGTH = 12;
```

Comment:
I'm assuming this is just an arbitrary choice for this test but might be nice to leave a comment about how this is actually determined in practice since its sometimes useful to look at these tests to understand the protocol

Reply:
thanks, added a comment